### PR TITLE
Add write query example by enabling the user to vote in the movie

### PR DESCRIPTION
**Pull request by bigmontz**
The goal of this change is provide a simple example of how to run a simple write query to the repository.

### CODE REVIEW

**Review by fbiville — CHANGES_REQUESTED** [review]
The e2e tests seem to run fine on my machine but:
```go
2021/06/01 15:57:20 http: panic serving 127.0.0.1:64422: interface conversion: interface {} is nil, not int64
goroutine 64 [running]:
net/http.(*conn).serve.func1(0xc0000961e0)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:1824 +0x153
panic(0x12a8040, 0xc000650420)
	/usr/local/Cellar/go/1.16.4/libexec/src/runtime/panic.go:971 +0x499
main.searchHandlerFunc.func1.1(0x13527c0, 0xc0000c0160, 0x0, 0x0, 0x0, 0x0)
	/Users/fbiville/workspace/neo4j-movies-go/server.go:108 +0x745
github.com/neo4j/neo4j-go-driver/v4/neo4j.(*session).runRetriable(0xc00007c0c0, 0x1, 0xc00050c0b0, 0x0, 0x0, 0x0, 0x10, 0x12b3b60, 0xc0001d4201, 0xc00050c0b0)
	/Users/fbiville/go/pkg/mod/github.com/neo4j/neo4j-go-driver/v4@v4.3.0/neo4j/session.go:285 +0x3ff
github.com/neo4j/neo4j-go-driver/v4/neo4j.(*session).ReadTransaction(0xc00007c0c0, 0xc00050c0b0, 0x0, 0x0, 0x0, 0x12f39bd, 0x6, 0x0, 0x0)
	/Users/fbiville/go/pkg/mod/github.com/neo4j/neo4j-go-driver/v4@v4.3.0/neo4j/session.go:333 +0x65
main.searchHandlerFunc.func1(0x1351f10, 0xc00055c120, 0xc000204200)
	/Users/fbiville/workspace/neo4j-movies-go/server.go:88 +0x2df
net/http.HandlerFunc.ServeHTTP(0xc000194d80, 0x1351f10, 0xc00055c120, 0xc000204200)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:2069 +0x44
net/http.(*ServeMux).ServeHTTP(0xc0001d0200, 0x1351f10, 0xc00055c120, 0xc000204200)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:2448 +0x1ad
github.com/daaku/go%2ehttpgzip.NewHandler.func1(0x1352180, 0xc0004a61c0, 0xc000204200)
	/Users/fbiville/go/pkg/mod/github.com/daaku/go.httpgzip@v0.0.0-20130226210307-fd996995061c/httpgzip.go:39 +0x282
net/http.HandlerFunc.ServeHTTP(0xc0001a2150, 0x1352180, 0xc0004a61c0, 0xc000204200)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:2069 +0x44
net/http.serverHandler.ServeHTTP(0xc0001fc000, 0x1352180, 0xc0004a61c0, 0xc000204200)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:2887 +0xa3
net/http.(*conn).serve(0xc0000961e0, 0x1352600, 0xc00009a300)
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:1952 +0x8cd
created by net/http.(*Server).Serve
	/usr/local/Cellar/go/1.16.4/libexec/src/net/http/server.go:3013 +0x39b
```

**Comment by fbiville** [comment]
```go
voteCount, ok := votes.(int64)
if !ok {
     voteCount = 0
}
```
WDYT?

**Comment by bigmontz** [comment]
> ```go
> voteCount, ok := votes.(int64)
> if !ok {
>      voteCount = 0
> }
> ```
> 
> WDYT?

Adressed

**Review by fbiville — APPROVED** [review]

File: server.go [hunk 1]
```diff
@@ -168,6 +178,43 @@ func movieHandlerFunc(driver neo4j.Driver, database string) func(http.ResponseWr
...
+				return nil, err
+			}
+			var summary, _ = result.Consume()
+			var voteResult VoteResult
+			voteResult.Updates = summary.Counters().PropertiesSet()
+
+			return voteResult, nil
+		})
+		if err != nil {
+			log.Println("error votigin movie:", err)
```
Comment (fbiville) [inline server.go]:
```suggestion
			log.Println("error voting for movie:", err)
```

**Review by fbiville — APPROVED** [review]

**Pull request merged**